debezium-core/src/main/java/io/debezium/pipeline/source/spi/ChangeTableResultSet.java

```diff
@@ -100,4 +110,6 @@ public String toString() {
     protected abstract int getOperation(ResultSet resultSet) throws SQLException;
 
     protected abstract T getNextChangePosition(ResultSet resultSet) throws SQLException;
+
+    protected abstract boolean isNewTransaction() throws SQLException;
```

Comment:
Could you add a JavaDoc comment describing the semantics; in particular, what's "new" in this context?

Reply:
Added.